app/jobs/document_proofing_job.rb

```diff
@@ -9,7 +9,6 @@ def perform(
     result_id:,
     encrypted_arguments:,
     trace_id:,
-    liveness_checking_enabled:,
```

Comment:
you'll either need a `**_rest` or to default this to `nil` because this is a background job

I'd recommend the `nil` approach and then after a deploy, another PR to remove it

Reply:
good catch, thank you